src/test/kotlin/lotto/ManualTicketCountValidatorTest.kt

```diff
@@ -0,0 +1,25 @@
+package lotto
+
+import lotto.model.ManualTicket
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertDoesNotThrow
+import org.junit.jupiter.api.assertThrows
+
+class ManualTicketCountValidatorTest {
+    @Test
+    fun `manual ticket count is a greater than possible tickets throws exception`() {
+        val exception =
+            assertThrows<IllegalArgumentException> {
+                ManualTicket.checkTicketCount(4, 2200)
+            }
+        assertThat("Manual tickets cannot be more than the possible purchase").isEqualTo(exception.message)
```

Comment:
```suggestion
        assertThat(exception).hasMessage("Manual tickets cannot be more than the possible purchase")
```

There's a nice tool in AssertJ for this situation :)

Reply:
Thank you that's a very useful function!